aslam_cv_calibration/src/target-algorithms.cc

```diff
@@ -0,0 +1,51 @@
+#include "aslam/calibration/target-algorithms.h"
+
+#include <Eigen/Core>
+#include <aslam/geometric-vision/pnp-pose-estimator.h>
+
+namespace aslam {
+namespace calibration {
+
+bool estimateTargetTransformation(
```

Comment:
Why not using a single function with [default arguments](http://en.cppreference.com/w/cpp/language/default_arguments)?

Reply:
I think the preferred ASL style is having multiple functions instead of default args.

Reply:
OK lets stick with this then ;-)